test/reifyhealth/specmonstah/core_test.cljc

```diff
@@ -67,12 +67,12 @@
                  (lat/add-attr :u0 :index 0)
                  (lat/add-attr :u0 :query-term [3])
                  (lat/add-attr :u0 :ent-type :user)
-                 
+
```

Comment:
My editor auto removes whitespace at the end of lines. I can revert this if it's really distracting